src/tutorials/lua_basics_lightbulb.md

```diff
@@ -25,103 +25,184 @@ This involves turning on and off a light switch to illuminate a light bulb.
+
+Type the following into the next line of `LightSwitchToggle`:
 
 ```lua
-local switch = script.parent
+local Trigger = script:GetCustomProperty("Trigger"):WaitForObject()
 ```
 
-- `local` is a **keyword** that indicates that you are making a variable. We use the term *local* here because it will only be accessible from this script rather than being accessible from external scripts, or globally.
-- `switch` is the variable name. You can name it anything but it's important to create variables with self-explanatory names so our scripts are easy to read and understand.
-- `script.parent` refers to the script's parent - the group or folder the script is placed in. In this case it refers to the **Switch** group. If you wanted to reference the entire **Light Switch & Bulb template**, for example, you would use `script.parent.parent.parent`.
+The script should now look like:
+
+```lua
+local Switch = script:GetCustomProperty("Switch"):WaitForObject()
+local Light = script:GetCustomProperty("Light"):WaitForObject()
+local Trigger = script:GetCustomProperty("Trigger"):WaitForObject()
+```
 
 ## Finding the Switch's Rotation
 
-To figure out how to rotate the switch, you will move it around in game, and take note of the **Rotation** displayed in the switch's **Properties** menu.
+To figure out how to rotate the switch, you will rotate it around in game, and take note of the **Rotation** property displayed in the switch's **Properties** window.
 
 ### Select the Switch
 
-Click on the **Switch (networked)** group in the **Hierarchy** to select it. Note that you will need to select the entire group, not just the cube inside of it, because that is what `switch` variable refers to.
+Click on the **Switch (networked)** group in the **Hierarchy** to select it. Note that you will need to select the entire group, not just the cube inside of it, because the entire group is what the `Switch` variable we defined earlier refers to.
 
 ![Selected Switch Object](../img/LightBulb/updates/SelectedSwitchObject.png){: .image-frame .center loading="lazy" }
 
 ### Find the Switch's Current Rotation
 
-1. Turn on **Rotation** mode by clicking the ![Rotation](../img/EditorManual/icons/Icon_TransformRotation.png) button, or pressing ++E++
+Turn on **Rotation** mode by clicking the ![Rotation](../img/EditorManual/icons/Icon_TransformRotation.png) button, or pressing ++E++
 { .image-inline-text .image-background }
 
-    ![Switch Rotation Mode](../img/LightBulb/updates/SwitchRotationMode.png){: .image-frame .center loading="lazy" }
+![Switch Rotation Mode](../img/LightBulb/updates/SwitchRotationMode.png){: .image-frame .center loading="lazy" }
 
-2. Look at the **Properties** window. In the **Transform** section, you should see a property called **Rotation**, with an **X**, **Y**, and **Z** value.
+!!! hint
+    Learning the editor bindings, such that ++E++ switches to **Rotation** mode can help productivity and speed while using the Core editor.
```

Comment:
- Link this to editor_shortcuts.md (double check that that's the right name)
- Take out "Learning" and change "editor bindings" to "keyboard shortcuts".